contracts/PaymentModule.sol

```diff
@@ -132,6 +143,18 @@ contract PaymentModule is StorageStructure, Ownable {
         } else return 0;
     }
 
+    function checkRegisterPayment(
```

Comment:
@ykamber99 do you want to check if a payment was registered? If yes, I would use a boolean variable as the return value. True if the payment is registered, and false if not.

Reply:
Yes!

Reply:
@ykamber99 then make that little code change and i will merge

Reply:
and update the test too, please

Reply:
I have updated the function and added a related test.